platform/ios/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 ### Packaging
 
+* Added the `Mapbox-iOS-SDK-stripped` build flavor, featuring fewer debug symbols. Removed the `Mapbox-iOS-SDK-symbols` build. ([#13504](https://github.com/mapbox/mapbox-gl-native/pull/13504))
```

Comment:
shouldn't we mention that the "plain" build now features full debugging information?

Reply:
How about...

> Added the `Mapbox-iOS-SDK-stripped` build flavor, featuring fewer debug symbols. Regular framework binaries are no longer stripped of debug symbols and the `Mapbox-iOS-SDK-symbols` build has been retired.

Reply:
This sounds good to me.